CMake/External_VXL.cmake

```diff
@@ -0,0 +1,128 @@
+# ZLIB
+add_package_dependency(
+  PACKAGE VXL
+  PACKAGE_DEPENDENCY ZLib
+  PACKAGE_DEPENDENCY_ALIAS ZLIB
+  )
+
+# JPEG
+add_package_dependency(
+  PACKAGE VXL
+  PACKAGE_DEPENDENCY libjpeg-turbo
+  PACKAGE_DEPENDENCY_ALIAS JPEG
+  )
+  
+# libtiff
+add_package_dependency(
+  PACKAGE VXL
+  PACKAGE_DEPENDENCY libtiff
+  PACKAGE_DEPENDENCY_ALIAS TIFF
+  )
+
+if (WIN32)
+  set (jpeg_lib_name "jpeg.lib")
+  set (zlib_library_name "zlib.lib")
+elseif (APPLE)
+  set (jpeg_lib_name "libjpeg.dylib")
+  set (zlib_library_name "libzlib.dylib")
+else()
+  set (jpeg_lib_name "libjpeg.so")
+  set (zlib_library_name "libzlib.so")
+endif()
+
+if(NOT JPEG_FOUND)
+  set(JPEG_INCLUDE_DIR ${fletch_BUILD_INSTALL_PREFIX}/include)
+  set(JPEG_LIBRARY ${fletch_BUILD_INSTALL_PREFIX}/lib/${jpeg_lib_name})
+endif()
+
+set(VXL_ARGS_CONTRIB
+  -DBUILD_BRL:BOOL=OFF
+  -DBUILD_MUL_TOOLS:BOOL=OFF
+  -DBUILD_PRIP:BOOL=OFF
+  )
+
+# Not using ffmpeg right now
+#set(VXL_ARGS_VIDL
+#  -DFFMPEG_CONFIG:FILEPATH=IGNORE
+#  -DFFMPEG_INCLUDE1_DIR:PATH=IGNORE
+#  -DFFMPEG_INCLUDE2_DIR:PATH=IGNORE
+#  -DFFMPEG_avcodec_LIBRARY:PATH=IGNORE
+#  -DFFMPEG_avformat_LIBRARY:PATH=IGNORE
+#  -DFFMPEG_avutil_LIBRARY:PATH=IGNORE
+#  -DFFMPEG_swscale_LIBRARY:PATH=IGNORE
+#  )
+
+if(UNIX)
+  set(VXL_ARGS_V3P
+    -DVXL_USING_NATIVE_ZLIB:BOOL=TRUE
```

Comment:
We have some ZLib stuff up above. We would rather pass our ZLib to VXL if we're building.